projectq/backends/_sim/_simulator.py

```diff
@@ -97,7 +98,8 @@ def is_available(self, cmd):
         """
         if (cmd.gate == Measure or cmd.gate == Allocate
            or cmd.gate == Deallocate
-           or isinstance(cmd.gate, BasicMathGate)):
+           or isinstance(cmd.gate, BasicMathGate)
+           or isinstance(cmd.gate, TimeEvolution)):
```

Comment:
Never mind. Users can use an instruction filter engine to decompose Hamiltonians with only a single term if the decomposition to one and two-qubit gates is faster to simulate.